### ISSUE TITLE
Version Tag needed by Module Registries

### ISSUE THREAD

**Issue by backspaces** [KhronosGroup/WebGLDeveloperTools#2]
TL;DR: God I hate to ask for yet-another favor.  But .. could we place a sem-ver version on webgl-debug?

Details:
The maintainers of the JSPM registry like to have version tags so that they can deal with varying version within a complex set of dependencies.  They may be able to use the commit SHA but would much prefer a version tag.  Even tho webgl-debug hasn't changed in almost 4 years!

The good news is, if you can do this, webgl-debug will have a lovely registry that uses HTTP2/SPDY for mind-blowing micro-module speeds.

Micro modules are becoming popular in webgl via the stack.gl suite of modules.  They even got toji's superb matrix library and refactored into seriously tiny modules.  I haven't tried it yet but looks like a good idea. I wish shaders supported modules, but "glsl-ify" is the next best thing.

**Comment by kenrussell** [comment]
Could you submit a pull request adding this? I have no experience with the versioning mechanism you're describing. Thanks.

See https://www.khronos.org/webgl/wiki/Using_Github_To_Contribute for instructions on contributing to the WebGL repository...replace that repository with WebGLDeveloperTools.

**Comment by backspaces** [comment]
Blush! Now I have to learn how! But sure, I'd love to do it and will figure
it out, likely easy.
